Runtime/Antlr3.Runtime/CommonToken.cs

```diff
@@ -250,11 +250,13 @@ public override string ToString()
             return "[@" + TokenIndex + "," + start + ":" + stop + "='" + txt + "',<" + type + ">" + channelStr + "," + line + ":" + CharPositionInLine + "]";
         }
 
+#if !PORTABLE
```

Comment:
The reason why I had to comment it (actually commenting the attribute is good enough) is that when I tried to used the PCL in a desktop unit test project, I got a `TypeLoadException` similar to what is described at http://stackoverflow.com/questions/20666731/typeloadexception-when-using-pcl-in-net-application-if-called-class-contains-o/20872983#20872983. So I think it will not work in a desktop project.